tests/suites/test_suite_ecp.function

```diff
@@ -955,7 +955,7 @@ exit:
 }
 /* END_CASE */
 
-/* BEGIN_CASE */
+/* BEGIN_CASE depends_on:!MBEDTLS_USE_PSA_CRYPTO */
```

Comment:
Same comment as `ecp.h`: I don't think we should be touching `test_suite_ecp` until #7390